### PR TITLE
Improve SCHEDULE description

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Make it easier for openQA users and newcomers to understand the SCHEDULE parameter.

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Supported variables per backend
 Variable;Values allowed;Default value;Explanation
 INCLUDE_MODULES;string;;comma separated names or fullnames of test modules to be included while excluding all that do not match, e.g. "boot,mod1"
 EXCLUDE_MODULES;string;;comma separated names or fullnames of test modules to exclude. Can be combined with INCLUDE_MODULES and has precedence, e.g. to additionally exclude modules based on an include-list
-SCHEDULE;string;;comma separated list of relative paths to test modules within CASEDIR without the implicit file extension '.pm' to be scheduled instead of evaluating a schedule from the test distributions main.pm file, e.g. "boot,console/mod1"
+SCHEDULE;string;;comma separated list of relative paths to test modules within *CASEDIR* without the implicit file extension '.pm' to be scheduled ignoring the distribution's main.pm file. _e.g. for https://github.com/os-autoinst/os-autoinst-distri-opensuse: "SCHEDULE=tests/boot/boot_to_desktop.pm,tests/console/systemd_testsuite"_.
```
[the remark this answers is not in the document]
Comment (SergioAtSUSE) [reply doc/backend_vars.asciidoc]: The _ is to put the example in italic. My asciidoc viewer shows it properly.

File: doc/backend_vars.asciidoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ Supported variables per backend
 Variable;Values allowed;Default value;Explanation
 INCLUDE_MODULES;string;;comma separated names or fullnames of test modules to be included while excluding all that do not match, e.g. "boot,mod1"
 EXCLUDE_MODULES;string;;comma separated names or fullnames of test modules to exclude. Can be combined with INCLUDE_MODULES and has precedence, e.g. to additionally exclude modules based on an include-list
-SCHEDULE;string;;comma separated list of relative paths to test modules within CASEDIR without the implicit file extension '.pm' to be scheduled instead of evaluating a schedule from the test distributions main.pm file, e.g. "boot,console/mod1"
+SCHEDULE;string;;comma separated list of relative paths to test modules within *CASEDIR* without the implicit file extension '.pm' to be scheduled overriding the distribution's main.pm file. _e.g. for https://github.com/os-autoinst/os-autoinst-distri-opensuse: "SCHEDULE=tests/boot/boot_to_desktop,tests/console/mod1"_.
```
Comment (okurz) [inline doc/backend_vars.asciidoc]: sorry but "overriding" still sounds like that the file would be ignored. What is wrong with my previous way of phrasing?

Comment (okurz) [reply doc/backend_vars.asciidoc]: yes, works for me :)

**Comment by Soulofdestiny** [comment]
LGTM

File: doc/backend_vars.asciidoc [hunk 3]
```diff
@@ -8,7 +8,7 @@ Supported variables per backend
 Variable;Values allowed;Default value;Explanation
 INCLUDE_MODULES;string;;comma separated names or fullnames of test modules to be included while excluding all that do not match, e.g. "boot,mod1"
 EXCLUDE_MODULES;string;;comma separated names or fullnames of test modules to exclude. Can be combined with INCLUDE_MODULES and has precedence, e.g. to additionally exclude modules based on an include-list
-SCHEDULE;string;;comma separated list of relative paths to test modules within CASEDIR without the implicit file extension '.pm' to be scheduled instead of evaluating a schedule from the test distributions main.pm file, e.g. "boot,console/mod1"
+SCHEDULE;string;;comma separated list of relative paths to test modules within *CASEDIR* to be scheduled. _e.g. "SCHEDULE=boot/boot_to_desktop,console/mod1"_.
```
Comment (okurz) [inline doc/backend_vars.asciidoc]: That's not right. Why did you delete the part about the extension? Despite what os-autoinst-distri-opensuse does, isotovideo in this case does the same. Using more asciidoc formating is nice but then I would recommend to do that in the whole file for more variables. What you should focus on here is the explanation itself. That has not improved in the latest version of your change

**Comment by SergioAtSUSE** [comment]
I am tired of this PR.

The documentation is wrong, because os-autoinst doesn't remove the .pm.
